tools/inkscape_extension/visicut_export.py

```diff
@@ -71,14 +69,16 @@ def get_single_instance_port():
                 sys.stderr.write("Warning: Cannot determine session ID. please report this.\n")
     return port
 
-# if Visicut or Inkscape cannot be found, change these lines here to VISICUTDIR="C:/Programs/Visicut" or wherever you installed it.
+# if Visicut or Inkscape cannot be found in PATH, change these lines here to VISICUTDIR="C:/Programs/Visicut" or wherever you installed it.
```

Comment:
The intended meaning was: "If the extension tells you that it can not find VisiCut or Inkscape, change these lines to where your VisiCut or Inkscape is." The target audience does not necessarily know what "PATH" is.